be1-go/message/messagedata/chirp_broadcast.go

```diff
@@ -8,3 +13,18 @@ type ChirpBroadcast struct {
 	Channel   string `json:"channel"`
 	Timestamp int64  `json:"timestamp"`
 }
+
+// Verify verifies that the ChirpBroadcast message is correct
+func (message ChirpBroadcast) Verify() error {
```

Comment:
I don't see where the Verify functions are called in the code. Should you add the actual verification in this PR ?

Reply:
Okay, I've added the actual verification in this PR. I merged my PR with the reaction channel into this one in order to do this.